src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double smallerSide;
+    private double biggerSide;
```

Comment:
if you name like so, we need to validate values so they really will be smaller and bigger
```suggestion
    private double bottomSide;
    private double topSide;
```